django/bosscore/public_channels.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 The Johns Hopkins University Applied Physics Laboratory
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+os.chdir(os.path.abspath(os.path.dirname(__file__)))
```

Comment:
I'm kind of worried about changing the working directory. Instead can you just provide the absolute path to `open()`?

```
channels_path = os.path.join(os.path.abspath(os.path.dirname(__file__)), 'public_channels.txt')
```

Reply:
I like it!